docs/sources/plugins/_index.md

```diff
@@ -4,5 +4,48 @@ type = "docs"
 [menu.docs]
 name = "Plugins"
 identifier = "plugins"
-weight = 8
+weight = 1
 +++
+
+# Plugins
+
+In addition to the wide range of visualizations and data sources that are available immediately after you install Grafana, you can extend your Grafana experience through _plugins_.
+
+You can [install]({{< relref "./installation.md" >}}) one of the plugins built by the Grafana community, or [build one yourself]({{< relref "./developing/development.md" >}}).
```

Comment:
Rather than linking to installation.md, I recommend linking to https://grafana.com/grafana/plugins?orderBy=weight&direction=asc

The Install plugins instructions basically say "go to this page and follow the directions"

Reply:
I feel silly for asking, but why do why have a docs page for installing plugins then? :)